lib/src/utils.dart

```diff
@@ -247,3 +247,13 @@ Stream<ProcessSignal> getSignals() => Platform.isWindows
     ? ProcessSignal.sigint.watch()
     : StreamGroup
         .merge([ProcessSignal.sigterm.watch(), ProcessSignal.sigint.watch()]);
+
```

Comment:
Suggestion: "utils" files tends to become the kitchen sink. How about sample.dart or a collection of presentational string APIs.

Reply:
Moving to `messages.dart`.